src/main/java/com/e_commerce/backend/controllers/CartController.java

```diff
@@ -0,0 +1,83 @@
+package com.e_commerce.backend.controllers;
+
+import com.e_commerce.backend.dtos.requests.AddProductToCartDto;
+import com.e_commerce.backend.dtos.requests.CreateCartDto;
+import com.e_commerce.backend.dtos.requests.UpdateCartItemQuantity;
+import com.e_commerce.backend.dtos.responses.ApiResponseDto;
+import com.e_commerce.backend.dtos.responses.CartDetailsResponseDto;
+import com.e_commerce.backend.dtos.responses.CartItemResponseDto;
+import com.e_commerce.backend.models.Cart;
+import com.e_commerce.backend.models.CartItem;
+import com.e_commerce.backend.services.CartService;
+import com.e_commerce.backend.utils.ResponseUtil;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/cart")
+public class CartController {
+    private final CartService cartService;
+
+    @PostMapping("")
+    @ResponseStatus(HttpStatus.CREATED)
+    public ApiResponseDto<Cart> createCart(@RequestBody CreateCartDto createCartDto) {
+        Cart cart = this.cartService.createCart(createCartDto);
+
+        return ResponseUtil.success(cart, "Cart created successfully");
+    }
+
+    @GetMapping("/{cart_id}")
+    @ResponseStatus(HttpStatus.OK)
+    public ApiResponseDto<CartDetailsResponseDto> getCartById(@PathVariable("cart_id") String cartId) {
+        CartDetailsResponseDto cartDetailsResponseDto = this.cartService.fetchCartDetailsById(cartId);
+
+        return ResponseUtil.success(cartDetailsResponseDto,"Cart details have been fetched");
```

Comment:
Please remove this string and reference it from a class

Reply:
@temi-tope Done